eclipse-collections-api/src/main/java/org/eclipse/collections/api/factory/bag/strategy/MutableHashingStrategyBagFactory.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2016 Goldman Sachs.
```

Comment:
@bhav0904, actually you are the copyright owner of the files you created. Since you're contributing as an individual, you can change the line to your name. Same case for other files you newly created. 

 * Copyright (c) 2016 YOUR NAME.

Refer to https://www.eclipse.org/legal/copyrightandlicensenotice.php.